src/frontend/src/icp/types/ic-send.ts

```diff
@@ -5,6 +5,7 @@ import type { TransferParams } from '$lib/types/send';
 export type IcTransferParams = Pick<TransferParams, 'amount' | 'to'> & {
 	identity: OptionIdentity;
 	progress: (step: ProgressStepsSendIc) => void;
+	ckEthMaxTransactionFee?: bigint;
```

Comment:
I suggest to rename this parameter for something more meaningful about it's usage - i.e. the fee are used as amount of the tranfer approval in the conversion of ckErc20. That way the parameter name would already gives us a hint of its usage.

Reply:
You know what I mean?

Reply:
I think I understood! It shouls say: max ckETH fees burned to cover Ethereum fees in case of ckERC20

What about `ckErc20MaxEthereumFee`? Or very verbose "ckErc20EthereumFeeToBeApproved"?

Reply:
Or `convertCkErc20ToErc20Fees` or else?

What I mean is also making clear that this variable is particularly use only for conversion of ckErc20 to Erc20

`ckErc20...` might also be interpreted as some fee for a simplem ckErc20 transfer

Does that make sense?

Reply:
ah! right! 

`ckErc20ToErc20EthereumFees`?

I think it is important to specify somewhere that they are ckETH or ETH-related? or not? if not, what you suggested is ok `convertCkErc20ToErc20Fees`

or even `convertCkErc20ToErc20MaxFees`

Reply:
Works for me

Reply:
done